cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -10,6 +10,11 @@ describe('FluxQueryBuilder', () => {
           cy.setFeatureFlags({
             newDataExplorer: true,
           }).then(() => {
+            const writeData = []
+            for (let i = 0; i < 30; i++) {
+              writeData.push(`ndbc,air_temp_degc=${i}_degrees station_id=${i}`)
```

Comment:
```suggestion
              writeData.push(`ndbc,air_temp_degc=${i}_degrees station_id_${i}=${i}`)
```

Might want to create 30 different measurements so that "Load more" button can be tested properly later in line 151 and 160

Reply:
op. i see the confusion. ari removed the testing of the load more functionality for fields, but also added it for tag values?

Reply:
Got it! Then disregard my suggestion 😅